README.md

```diff
@@ -22,121 +22,112 @@ refreshing the code and making it available for easy reuse by others._
 
 ## Architecture
 The model is written in Python, using PyTorch for the CNN. We provide 3 separate
-"stages", which are run using different commands and arguments:
+"steps", which are run using different commands and arguments:
 
 * data processing: downloads part of CM2.6 dataset and processes
 * model training: train model on processed data
 * model testing: tests the trained model on an unseen region
 
-For more details on each of the stages, see the [`docs`](docs/) directory.
+For more details on each of the steps, see the [`docs`](docs/) directory.
 
 ## Usage
 ### Dependencies
-Python 3 is required.
+Python 3.9 or newer is required.
 
 #### Python
 With `pip` installed, run the following in the root directory:
 
     pip install -e .
 
-To avoid any conflicts with local packages, we recommend using a virtual
-environment. In the root directory:
+We suggest using a virtual environment. In the root directory, create like so:
 
-    virtualenv venv
-    source venv/bin/activate
+    python -m venv venv
 
-See [virtualenv docs](https://virtualenv.pypa.io/en/latest/) for more details.
+Then load with `source venv/bin/activate`.
 
-Alternatively, if you are using python to manage virtual environments using the `venv` module, then the first line above can be replaced by `python -m venv venv` (where the second `venv` is the virtual environment name).
+Alternatively, you may use the
+[virtualenv](https://virtualenv.pypa.io/en/latest/) tool, in which case you
+should replace the first command with `virtualenv venv`.
 
 *(An alternate `pyproject.toml` file is provided for building with
 [Poetry](https://python-poetry.org/). To use, rename `pyproject-poetry.toml` to
 `pyproject.toml` (overwriting the existing file) and use Poetry as normal. Note
 that the Poetry build is not actively supported-- if it fails, check that the
 dependencies are up to date with the setuptools `pyproject.toml`.)*
 
-#### System
-Some graphing code uses cartopy, which requires [GEOS](https://libgeos.org/). To
```

Comment:
are we sure that we do not need this anymore?

Reply:
We don't for Python 3.10 and above. I've changed this to say "please use Python >3.10 or go install this somehow".